dune-project

```
@@ -1,2 +1,77 @@
 (lang dune 2.3)
+
 (name unison)
+
+(version dev)
+
+(generate_opam_files false)
```

Set  this to `true` so that `dune build` updates the OPAM files as well.
```suggestion
(generate_opam_files true)
```

---

For now, I'm ok with not automatically overwriting checked-in files.